File: Offices/Offices.Application/Models/DoctorModel.cs
﻿using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Offices.Application.Models
{
    public record DoctorModel(Guid Id, string FirstName, string LastName, string? MiddleName, Guid OfficeId, string Status);
}

File: react-client/src/models/doctors/doctorModel.ts
import { DoctorStatus } from "../../entities/enums/doctorStatus";

export default interface DoctorModel {
  id: string,
  firstName: string,
  lastName: string,
  middleName?: string,
  specializationId: string,
  accountId: string,
  officeId: string,
  careerStartYear: Date,
  dateOfBirth: Date,
  Status: DoctorStatus
}